src/brouwers/kitreviews/templates/kitreviews/add_review.html

```diff
@@ -28,7 +28,7 @@
                 {% include "form_in_table.html" %}
             {% endwith %}
             <div class="center">
-                <input type="button" value="{% trans 'Find kits' %}" class="submit-button" />
+                <input type="button" value="{% trans 'Find kits' %}" class="submit-button"/>
```

Comment:
templates/styling is/are a mess in general, feel free to base it on `base_html5.html` and improve the styling, or come up with mock-ups if you want to. I can tackle it later as well if you prefer that.